### ISSUE TITLE
Quark label report does not work any longer UnboundLocalError

### ISSUE THREAD

**Issue by cryptax** [quark-engine/quark-engine#304]
quark with option `-l detailed` or `-l max` crashes with `UnboundLocalError`. This used to work, so probably a _regression_?

```
# quark -a smsthief.apk -l detailed
...
           \__>          \/           \/ v22.1.1
    
                An Obfuscation-Neglect Android Malware Scoring System
                
Requested API level 30 is larger than maximum we have, returning API level 28 instead.
100%|██████████████████████████████████████████████████████████████████████████████████████████| 181/181 [00:43<00:00,  4.18it/s]
[*] Total Label found: 34
[*] Rules with label which max confidence >= 80%: 12
Traceback (most recent call last):
  File "/usr/local/bin/quark", line 8, in <module>
    sys.exit(entry_point())
  File "/usr/local/lib/python3.8/dist-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.8/dist-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.8/dist-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/quark/cli.py", line 304, in entry_point
    data.show_label_report(rule, all_labels, label)
  File "/usr/local/lib/python3.8/dist-packages/quark/core/quark.py", line 650, in show_label_report
    self.quark_analysis.label_report_table.field_names = [
  File "/usr/local/lib/python3.8/dist-packages/prettytable.py", line 416, in _set_field_names
    if self._align and old_names:
UnboundLocalError: local variable 'old_names' referenced before assignment
```

**To Reproduce**
- Please get an APK. I tested with 2 totally different ones, and got the crash in both case, so I believe this does not depend on the APK.
- Run `quark -a your.apk -l detailed` or `quark -a your.apk -l max`

**Desktop (please complete the following information):**
 - OS: Docker container from *Ubuntu 20.04* (https://github.com/cryptax/androidre/blob/master/Dockerfile.re)
 - Version: `v22.1.1`
 - Python `3.8.10`

**Comment by pulorsok** [comment]
Hi @cryptax 

Thank you for reporting this issue!
We will start to investigate it. If there is anything new, I will keep you updated :)

**Comment by haeter525** [comment]
Hi @cryptax 

I tested on the container and got the same error as well. It seems like one of Quark's dependencies, prettytable, is outdated (version 0.7.2) and causes Quark to crash. I found version 1.0.0 is the minimal version that Quark can work.

Therefore, I will open a PR later to fix the incorrect version of prettytable. And we will release the latest Quark soon. Thank you for reporting this issue.
